casbin/internal_enforcer.py

```diff
@@ -22,6 +22,23 @@ def _add_policy(self, sec, ptype, rule):
 
         return rule_added
 
+    def _add_policies(self, sec, ptype, rules):
+        """adds rules to the current policy."""
+        for rule in rules:
+            rule_added = self.model.add_policy(sec, ptype, rule)
+            if not rule_added:
+                continue
+
+            if self.adapter and self.auto_save:
+                if self.adapter.add_policy(sec, ptype, rule) is False:
+                    continue
+
+                if self.watcher:
+                    self.watcher.update()
+
```

Comment:
refer to: https://github.com/casbin/casbin/blob/master/internal_api.go#L73

Reply:
Well, I make a small change to give a warning for this occasion